src/quickstart.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+OWNER=microshift-io
+REPO=microshift
+BOOTC_ARCHIVE=""
+
+LVM_DISK="/var/lib/microshift-okd/lvmdisk.image"
+VG_NAME="myvg1"
+
+TMPDIR="$(mktemp -d /tmp/microshift-okd-XXXXXX)"
+trap 'rm -rf ${TMPDIR} &>/dev/null' EXIT
+
+function download_bootc_image() {
+    local -r filter="$1"
+
+    pushd "${TMPDIR}" >/dev/null
+    local -r url=$(curl -s https://api.github.com/repos/$OWNER/$REPO/releases/latest \
+        | grep "browser_download_url" \
+        | cut -d '"' -f 4 \
+        | grep "${filter}")
+
+    if [ -z "${url}" ]; then
+        echo "ERROR: No packages found for '${filter}'"
+        return 1
+    fi
+
+    echo "Downloading '${url}'"
+    curl -L --progress-bar -O "${url}"
+    popd >/dev/null
+
+    BOOTC_ARCHIVE="${TMPDIR}/$(basename "${url}")"
+}
+
+function install_bootc_image() {
+    local -r image_file="$1"
+
+    echo "Installing '${image_file}'"
+    podman load -q -i "${image_file}"
+}
+
+function prepare_lvm_disk() {
+    local -r lvm_disk="$1"
+    local -r vg_name="$2"
+
+    if [ -f "${lvm_disk}" ]; then
+        echo "INFO: '${lvm_disk}' already exists. Reusing it."
+        return 0
+    fi
+
+    mkdir -p "$(dirname "${lvm_disk}")"
+    truncate --size=1G "${lvm_disk}"
+    losetup -f "${lvm_disk}"
+
+    local -r device_name="$(losetup -j "${lvm_disk}" | cut -d: -f1)"
+    vgcreate -f -y "${vg_name}" "${device_name}"
+}
+
+function run_bootc_image() {
+    local -r image_name="$1"
+
+    echo "Running '${image_name}'"
+    modprobe openvswitch || true
```

Comment:
I think name of the published image should indicate it's ovnk "edition".
We can handle this separately, I'd like to implement pushing the imagge to github registry, so we can think about proper tags

Reply:
This task would require more investigation and changes. 
Can you open an issue for this work in the repo?